types/react-calendar-heatmap/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for react-calendar-heatmap v1.6.1
+// Project: https://github.com/patientslikeme/react-calendar-heatmap
+// Definitions by: Keisuke Kan <https://github.com/9renpoto>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="react" />
+
+declare module "react-calendar-heatmap" {
```

Comment:
You don't need to wrap this in a `declare module` block. You can leave it at the top level.